### PR TITLE
Added some basic things for rebooting

### PR DESCRIPTION
**Pull request by sthewissen**
Changes proposed in this pull request:
- Issue templates based on Essentials.
- Quick README update.
- Some basic files included regarding the project.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Nuget Packages:
+- Affected Devices:
+
+### Screenshots
+
+<!-- If the issue is a visual issue, please include screenshots showing the problem if possible -->
+
+### Reproduction Link
+
+<!-- Please upload or provide a link to a reproduction case -->
```
[the remark this answers is not in the document]
Comment (sthewissen) [reply .github/ISSUE_TEMPLATE/bug-report.md:41]: Sounds good!

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: "[Bug] "
+labels: bug
+assignees: ''
+
+---
+
+<!-- Bug report best practices: https://github.com/xamarin/Essentials/wiki -->
```
[the remark this answers is not in the document]
Comment (sthewissen) [reply .github/ISSUE_TEMPLATE/bug-report.md]: We don't even have a Wiki tab atm, don't think I can enable it :(

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,8 +1,44 @@
...
-Fixes # .
+Describe your changes here. 
 
-Changes Proposed in this pull request:
--
--
-- 
+### Bugs Fixed ###
+
+- Related to issue #
```
[the remark this answers is not in the document]
Comment (sthewissen) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: I didn't even know about these hidden automation features tbh 🙃

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  - Android: <!-- The version of the Android SDK you are compiling against, e.g. 7.1 --> 
+  - UWP:  <!-- The version of the UWP SDK you are compiling against, e.g. 16299 --> 
+- Android Support Library Version: <!-- if applicable -->
+- Nuget Packages:
+- Affected Devices:
+
+### Screenshots
+
+<!-- If the issue is a visual issue, please include screenshots showing the problem if possible -->
+
```
Comment (AndreiMisiukevich) [inline .github/ISSUE_TEMPLATE/bug-report.md:38]:
```suggestion
### Reproduction imagery

<!-- If it is a visual issue, please include imagery (GIF/Screenshots) showing the problem -->

```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+name: Feature request
+about: Suggest an idea for CommunityToolkit
+title: "[Enhancement] YOUR IDEA!"
+labels: feature-request 
+assignees: ''
+
+---
+
+## Summary
+Please provide a brief summary of your proposal. Two to three sentences is best here.
```
Comment (AndreiMisiukevich) [inline .github/ISSUE_TEMPLATE/feature-request.md:11]:
```suggestion
<!-- Please provide a brief summary of your proposal. Two to three sentences are best here. -->
```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+In order to facilitate the new Shiny Button api, a bool is added to the Button class. This is done as a bool because it is simpler to data bind and other reasons...
+
+    var button = new Button ();
+    button.MakeShiny = true; // new API
+
+The MakeShiny API works even if the button is already visible.
+
+## Intended Use Case
+Provide a detailed example of where your proposal would be used and for what purpose.
```
Comment (AndreiMisiukevich) [inline .github/ISSUE_TEMPLATE/feature-request.md:27]:
```suggestion

<!-- Provide a detailed example of where your proposal would be used and for what purpose. -->
```

**Comment by AndreiMisiukevich** [comment]
@sthewissen I believe the target branch should be **reboot** as well (now **not-active** is selected)

**Comment by sthewissen** [comment]
@AndreiMisiukevich My idea was to rename **not-active** to **master** and current **master** to **legacy**. That way we have our starting point based on not-active while keeping all of the git history. We don't have all the rights to do this though.

**Pull request merged**